tests/Main.hs

```diff
@@ -4,11 +4,20 @@ module Main where
 import Effectful
 import Effectful.Log
 import Log.Backend.StandardOutput
+import qualified Data.Aeson.Types as Aeson
 
 main :: IO ()
 main = runEff $ do
   withStdOutLogger $ \logger -> do
     runLog "main" logger defaultLogLevel app
+  withStdOutLogger $ \logger -> do
+    let logEnv = LoggerEnv  { leLogger = logger
+                            , leComponent = "main"
+                            , leDomain = []
+                            , leData = [("testKey", Aeson.String "testValue") ]
+                            , leMaxLogLevel = defaultLogLevel
+                            }
+    runLogWithEnv logEnv app
```

Comment:
I think changing the test is not necessary. Especially if `runLog` is defined in terms of `runLogWithEnv`; See previous comment.

Reply:
:+1: